index.ts

```diff
@@ -155,6 +158,19 @@ for (const [key, data] of yamlEntries('features')) {
         }
     }
 
+    // Check that no BCD key is used twice until the meaning is made clear in
+    // https://github.com/web-platform-dx/web-features/issues/1173.
+    if (data.compat_features) {
+        for (const bcdKey of data.compat_features) {
+            const otherKey = bcdToFeatureId.get(bcdKey);
+            if (otherKey) {
+                throw new Error(`BCD key ${bcdKey} is used in both ${otherKey} and ${key}, which creates ambiguity for some consumers. Please see https://github.com/web-platform-dx/web-features/issues/1173 and help us find a good solution to allow this.`);
```

Comment:
I really like this message. 👍

Reply:
I'm glad you like it :) It's a bit weird to throw and error and still say that we'd like to allow it...